pkg/maps/lbmap/ipv4.go

```diff
@@ -56,7 +56,7 @@ var (
 				return nil, nil, err
 			}
 
-			return svcKey.ToNetwork(), svcVal.ToNetwork(), nil
+			return svcKey, svcVal.ToNetwork(), nil
```

Comment:
Instead of calling `svcVal.ToNetwork()` we should really call `svcVal.ToHost()` (the method is not yet implemented).